tui/mainModel.go

```diff
@@ -851,6 +827,15 @@ func (m *MainModel) prepopulateContainerSizeMapConcurrently() {
 	}
 }
 
+func (m MainModel) handleBackground(operation func(string) error) {
+	curItem := m.getSelectedItem()
+	if curItem == nil {
+		return
+	}
+	containerId := curItem.(dockerRes).getId()
+	m.possibleLongRunningOpErrorChan <- operation(containerId)
+}
+
```

Comment:
We should consider making this a pure function. 

i.e pass in `id` to the function as well. Cuz we cannot determine when a go routine will be scheduled to run. If the user somehow is selecting a different list item when `handleBackground` gets run, the `operation` will be run on that item instead. Which is not supposed to happen.
 
```suggestion
func (m MainModel) handleBackground(operation func(string) error, id string) {
	m.possibleLongRunningOpErrorChan <- operation(id)
}

```